examples/qasm/routing_example_circuit.qasm

```diff
@@ -1,804 +1,804 @@
-cx q[2],q[5];
-cx q[3],q[8];
-cx q[7],q[6];
-cx q[0],q[6];
-cx q[9],q[2];
-cx q[3],q[7];
-cx q[4],q[5];
-cx q[0],q[5];
-cx q[7],q[1];
-cx q[2],q[9];
-cx q[4],q[8];
-cx q[0],q[7];
-cx q[2],q[1];
-cx q[5],q[3];
-cx q[4],q[6];
-cx q[9],q[8];
-cx q[2],q[1];
-cx q[6],q[3];
-cx q[8],q[4];
-cx q[5],q[9];
-cx q[9],q[0];
-cx q[1],q[5];
-cx q[2],q[7];
-cx q[6],q[8];
-cx q[0],q[8];
-cx q[3],q[1];
-cx q[9],q[2];
-cx q[4],q[7];
-cx q[8],q[0];
-cx q[9],q[1];
-cx q[7],q[2];
-cx q[3],q[6];
-cx q[5],q[4];
-cx q[8],q[0];
-cx q[9],q[2];
-cx q[3],q[6];
-cx q[9],q[2];
-cx q[5],q[3];
-cx q[4],q[6];
-cx q[9],q[1];
-cx q[2],q[5];
-cx q[6],q[3];
-cx q[4],q[8];
-cx q[0],q[9];
-cx q[1],q[7];
-cx q[5],q[3];
-cx q[4],q[6];
-cx q[0],q[3];
-cx q[1],q[6];
-cx q[9],q[2];
-cx q[8],q[5];
-cx q[6],q[0];
-cx q[1],q[4];
-cx q[3],q[2];
-cx q[5],q[8];
-cx q[0],q[6];
-cx q[1],q[2];
-cx q[8],q[4];
-cx q[5],q[9];
-cx q[0],q[7];
-cx q[8],q[1];
-cx q[6],q[3];
-cx q[4],q[9];
-cx q[0],q[1];
-cx q[2],q[9];
-cx q[3],q[7];
-cx q[4],q[8];
-cx q[5],q[6];
-cx q[8],q[0];
-cx q[1],q[3];
-cx q[2],q[7];
-cx q[4],q[5];
-cx q[0],q[1];
-cx q[2],q[6];
-cx q[7],q[3];
-cx q[9],q[8];
-cx q[1],q[0];
-cx q[5],q[3];
-cx q[4],q[8];
-cx q[7],q[9];
-cx q[3],q[2];
-cx q[6],q[8];
-cx q[7],q[9];
-cx q[9],q[0];
-cx q[2],q[1];
-cx q[7],q[4];
-cx q[6],q[8];
-cx q[3],q[0];
-cx q[2],q[1];
-cx q[4],q[9];
-cx q[5],q[7];
-cx q[6],q[8];
-cx q[0],q[3];
-cx q[4],q[1];
-cx q[2],q[7];
-cx q[5],q[8];
-cx q[9],q[1];
-cx q[2],q[7];
-cx q[4],q[6];
-cx q[8],q[5];
-cx q[0],q[7];
-cx q[6],q[2];
-cx q[5],q[4];
-cx q[5],q[0];
-cx q[1],q[6];
-cx q[2],q[9];
-cx q[7],q[3];
-cx q[3],q[1];
-cx q[7],q[2];
-cx q[9],q[4];
-cx q[5],q[8];
-cx q[0],q[4];
-cx q[1],q[6];
-cx q[9],q[2];
-cx q[7],q[3];
-cx q[0],q[7];
-cx q[8],q[1];
-cx q[9],q[2];
-cx q[5],q[6];
-cx q[1],q[0];
-cx q[8],q[2];
-cx q[9],q[3];
-cx q[7],q[5];
-cx q[0],q[9];
-cx q[5],q[1];
-cx q[4],q[3];
-cx q[8],q[6];
-cx q[1],q[0];
-cx q[2],q[8];
-cx q[3],q[5];
-cx q[7],q[9];
-cx q[0],q[4];
-cx q[2],q[9];
-cx q[8],q[5];
-cx q[6],q[7];
-cx q[0],q[4];
-cx q[1],q[9];
-cx q[2],q[8];
-cx q[7],q[6];
-cx q[1],q[0];
-cx q[6],q[2];
-cx q[4],q[8];
-cx q[7],q[5];
-cx q[1],q[5];
-cx q[2],q[7];
-cx q[3],q[6];
-cx q[8],q[9];
-cx q[5],q[0];
-cx q[4],q[2];
-cx q[7],q[3];
-cx q[8],q[6];
-cx q[4],q[0];
-cx q[2],q[9];
-cx q[8],q[3];
-cx q[7],q[6];
-cx q[0],q[2];
-cx q[1],q[4];
-cx q[8],q[3];
-cx q[6],q[5];
-cx q[2],q[7];
-cx q[6],q[3];
-cx q[4],q[9];
-cx q[8],q[5];
-cx q[6],q[1];
-cx q[2],q[4];
-cx q[8],q[3];
-cx q[7],q[5];
-cx q[0],q[2];
-cx q[5],q[1];
-cx q[8],q[3];
-cx q[9],q[6];
-cx q[1],q[4];
-cx q[8],q[2];
-cx q[5],q[3];
-cx q[6],q[7];
-cx q[0],q[5];
-cx q[1],q[7];
-cx q[2],q[9];
-cx q[8],q[6];
-cx q[0],q[5];
-cx q[1],q[4];
-cx q[2],q[9];
-cx q[6],q[8];
-cx q[4],q[0];
-cx q[2],q[9];
-cx q[6],q[5];
-cx q[8],q[7];
-cx q[1],q[0];
-cx q[2],q[6];
-cx q[5],q[3];
-cx q[8],q[9];
-cx q[0],q[1];
-cx q[7],q[2];
-cx q[9],q[3];
-cx q[4],q[6];
-cx q[5],q[1];
-cx q[2],q[7];
-cx q[3],q[8];
-cx q[6],q[9];
+qreg qu[10];
```

Comment:
This is coupled to a change I need to make in the pytket-extensions which is that:
• The cirq_convert process_characteriastion method produces an Architecture with UnitID's labelled with 'q' but with 2 or 3d dimensional indexing
• The default qubit register for circuits is 'q' with 1d indexing
• When attempting to place circuit qubits by relabelling qubits, the circuit complains that it can't add a new qubit labelled with 'q' but with a different dimension index

The possible solutions are:
• Make the cirq_convert process_characterisation method not use 'q'
• Make the placement method remove known qubit registers before adding new ones (this will likely have hiccups)
• Allow qubits to be added with the same label but different dimensions of indexing (potentially cumbersome)